### DISCUSSION TITLE
Drag into script window shortcuts

### DISCUSSION THREAD

**Discussion by shak2** [godotengine/godot-proposals#9776] [Editor]
The following 4 shortcuts are not documented in the documentation:

- Drag node into script window.
- Hold CTRL and drag node into script window.
- Drag asset into script window.
- Hold CTRL and drag asset into script window.

Can we please add these to the docs.

Also, are there any others like these out there?

**Comment by theraot** [comment]
See https://github.com/godotengine/godot-docs/issues/4377
